### PR TITLE
Mark CBAPI as Release 1.7.0

### PR DESCRIPTION
**Pull request by abowersox-cb**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Tests have been added that prove the fix is effective or that the feature works.
- [x] New and existing tests pass locally with the changes.
- [x] Code follows the style guidelines of this project (PEP8, clean code).
- [x] Linter has passed locally and any fixes were made for failures.
- [x] A self-review of the code has been done.

## Pull request type

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes (not tied to bugs/features)
- [ ] Other (please describe):

## What is the ticket or issue number?

- Ticket Number: https://jira.carbonblack.local/browse/CBAPI-1774

- Issue Number: N/A

## Pull Request Description

Set version number to 1.7.0.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## How Has This Been Tested?

Documentation and config updates only, no code.

### CODE REVIEW

File: src/cbapi/__init__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 __author__ = 'Carbon Black Developer Network'
 __license__ = 'MIT'
 __copyright__ = 'Copyright 2018-2019 VMware Carbon Black'
```
Comment (avanbrunt-cb) [inline src/cbapi/__init__.py:8]: This copyright needs updated

File: docs/conf.py [hunk 2]
```diff
@@ -51,17 +51,17 @@
 
 # General information about the project.
 project = u'cbapi'
-copyright = u'2016-2019, Carbon Black Developer Network'
+copyright = u'2016-2020, VMware Inc.'
```
Comment (avanbrunt-cb) [inline docs/conf.py:54]: This should be `VMware Carbon Black`

File: src/cbapi/__init__.py (hunk 1 above)
Comment (abowersox-cb) [reply src/cbapi/__init__.py]: Wonder why it wasn't updated with Release 1.6.2? Oh well, updated it.

File: docs/conf.py (hunk 2 above)
Comment (abowersox-cb) [reply docs/conf.py]: Yes it should.

**Pull request merged**